services/gateway/shared/regex.js

```diff
@@ -34,6 +34,7 @@ const TOKEN_ID = /^\b[a-fA-F0-9]{16}\b$/;
 const BLS_KEY = /^\b[a-fA-F0-9]{96}\b$/;
 const PROOF_OF_POSSESSION = /^\b[a-fA-F0-9]{192}\b$/;
 const TRANSACTION = /^\b[0-9a-fA-F]+\b$/;
+const MODULE = /^(?:[a-z ]+,)*[a-z ]+$/;
```

Comment:
Please update the regex for MODULE name.
```suggestion
const MODULE = /^\b(?:[\w!@$&.]{1,32}|,)+\b$/;
```